### ISSUE TITLE
Tree items not accessible via Wooshy et al.

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#2734]
https://forums.zotero.org/discussion/98772/exposing-zotero-ui-on-macos

[Wooshy](https://wooshy.app/) doesn't work in Places in Firefox or the folder/items trees in Thunderbird, despite working in most of the rest of the UI, so maybe there's some more general issue here with a tree-like element. But in Finder, for example, it can focus a visible tree row. (A file row that's out of view can't be found.)

From #2733

**Comment by godbout** [comment]
cool thanks. didn't know about the Thunderbird thing. just did a quick test for a user. i'll check more and see if there's anything i can do on my side.

**Comment by adomasven** [comment]
Found some upstream bugs. We should wait and see if they get resolved. https://github.com/godbout/Wooshy.docs/issues/55

**Comment by godbout** [comment]
thank you!

**Comment by godbout** [comment]
so for the story, Firefox doesn't return proper data for the `AXOutline` role (it should return the list of visible rows but it returns an empty array instead). i've tweaked Wooshy so that it handles this case in some other ways, without any side effect for other apps (hopefully). the change is available in a16: https://github.com/godbout/Wooshy.docs/releases/tag/1.a16

not sure if this solves issues with Zotero yet tho.
